<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSOutboundEditor</class>
 <widget class="QWidget" name="SSOutboundEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ssPasswordTxt"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="ssMethodCombo">
     <item>
      <property name="text">
       <string notr="true">chacha20-ietf-poly1305</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rc4-md5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>aes-128-gcm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>aes-192-gcm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>aes-256-gcm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-128-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-192-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-256-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-128-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-192-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-256-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-128-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-192-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-256-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">bf-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>xchacha20-ietf-poly1305</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>salsa20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chacha20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chacha20-ietf</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>plugin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ssPluginTxt"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>plugin-opt</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="ssPluginOptTxt"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ssKeyTxt"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="selectFilePushButton">
     <property name="text">
      <string>选择Plugin文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
